spec/fixtures/fits_0.8.5_jp2.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
It handles 0.6 kinda well.  It was on my list for things to get working perfectly, but that can wait for another time.